tests/bluetooth/common/testlib/src/att_read.c

```diff
@@ -2,18 +2,22 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <autoconf.h>
```

Comment:
`autoconf.h` is a funny include, because of the way it's included. My Clang-tidy finds it under `zephyr/autoconf.h` after compilation, but it can also be included like this. 

I don't have a opinion of whether to use one or the other, but we probably should be somewhat consistent

Reply:
Looks like `zephyr/autoconf.h` is the preferred way.

https://github.com/zephyrproject-rtos/zephyr/blob/e1a0350d23706fa1201d5f37190c77141abe8197/Kconfig.zephyr#L1105-L1116

Reply:
The `-I` argument to gcc for `include/generated/zephyr` is placed before `include/generated` on the command line in the compilation database. That's probably why my clangd picked that one up.